### DISCUSSION TITLE
IMPORTANT: I AM ARCHIVING THIS REPO

### DISCUSSION THREAD

**Discussion by easytarget** [easytarget/esp32-cam-webserver#323] [Announcements]
As you might have noticed; I have taken no active interest in this since Covid ended.. I have not been idle; but busy with other projects and life in general.

#### I intend to wrap this up sensibly: in the `README`
With a clear description of the current status; and the important limitiations on the current code
* IDE and ESP ARDUINO versions required (the latter is really old; I see they are on v4.x now) 
* Platformio status
* Notes about hardware and current boards etc..

**Most importantly**; I think there are some forks of my code that are worth redirecting people to!
* I'd love to hear your suggestions for these!

I have started this moving by doing a big (and somewhat brutal) clean up of open issues today; and I'll be trimming the discussions here to just the most useful ones too.

I will be making the whole project 'archived' in gitHub soon. So that it is clear to people that it is essentially unsupported.

Comments below.

**Comment by BillyHGLGR** [comment]
Thank you for creating this in the first place.
